model/simulationtests/autosize_hvac.rb

```diff
@@ -1215,9 +1215,14 @@ def make_tes_coil(model)
     term.addToThermalZone(zn)
     vrf.addTerminal(term)
 
+  when 44
+    zoneHVACEvaporativeCoolerUnit = OpenStudio::Model::ZoneHVACEvaporativeCoolerUnit.new(model)
+    zoneHVACEvaporativeCoolerUnit.autosizeDesignSupplyAirFlowRate
+    zoneHVACEvaporativeCoolerUnit.addToThermalZone(zn)
+
```

Comment:
Added to the autosize_hvac! Which told me the method was not implemented correctly so did https://github.com/NREL/OpenStudio/pull/5326/commits/c9691108c1dcda4a974cc4a6be3cd87bf19bfae9